src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,202 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+
     @Override
     public void add(T value) {
+        Node<T> current = new Node<>(null, value, null);
+        if (head == null) {
+            head = current;
+            tail = current;
+        } else {
+            tail.next = current;
+            current.prev = tail;
+            tail = current;
+            tail.next = null;
+        }
     }
 
     @Override
     public void add(T value, int index) {
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/212983891-53de55be-b807-4474-839a-288b58bed8ba.png)
This is how this method should looks like. 
Create getNode and linkBefore methods